docs/rfcs/rfc-009-graphql-ui-poc.md

```diff
@@ -0,0 +1,112 @@
+# GraphQL Graph App PoC
+
+## Problem
+
+The [GraphQL Architect](https://grandstack.io/docs/graphql-architect-overview/#:~:text=GraphQL%20Architect%20is%20a%20graph,Neo4j%20Desktop%20Graph%20Apps%20Gallery) for usage in the Neo4j Desktop is due to be replaced. As an initial step a proof of concept of a successor shall be established.
+
+## Proposed Solution
+
+_Name suggestions:_  
+Explorer  
+Architect  
+Editor  
+UI  
+Frame  
+(Already taken: Studio, Playground, GraphiQL)
+
+### User audience
+
+The `@neo4j/graphql` lib's user audience are full stack developers who want a GraphQL API quickly without worrying about the database. This UI can further include developers wanting to experiment in general as well as experiment with/edit the introspected schema.
+
+### Workflow (Graph App)
+
+1. Login page
+    1. Specify the database credentials (connectURL, username, password)
```

Comment:
**Nice-to-have** (for future versions maybe): Store credentials in localStorage or similar?